backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlSchemaFieldFactory.java

```diff
@@ -455,18 +456,26 @@ private static void dropTables(
         .type(GraphQLList.list(outputSettingsMetadataType))
         .dataFetcher(
             dataFetchingEnvironment -> {
-              List<String> selectedKeys =
+              final List<String> selectedKeys =
                   dataFetchingEnvironment.getArgumentOrDefault(KEYS, new ArrayList<>());
               final boolean includePages =
                   selectedKeys.stream().anyMatch(selectedKey -> selectedKey.startsWith("page."));
-              return schema.getMetadata().getSettings().stream()
-                  .filter(
-                      setting ->
-                          selectedKeys.isEmpty()
-                              || selectedKeys.contains(setting.getKey())
-                              || (includePages && setting.getKey().startsWith("page.")))
-                  .map(entry -> Map.of("key", entry.getKey(), VALUE, entry.getValue()))
-                  .collect(Collectors.toList());
+
+              return Stream.concat(
+                      schema.getMetadata().getSettings().stream()
+                          .filter(
+                              setting ->
+                                  selectedKeys.isEmpty()
+                                      || selectedKeys.contains(setting.getKey())
+                                      || (includePages && setting.getKey().startsWith("page.")))
+                          .map(entry -> Map.of("key", entry.getKey(), VALUE, entry.getValue())),
+                      Stream.of(
+                          Map.of(
+                              "key",
+                              IS_OIDC_ENABLED,
```

Comment:
I would have expected that we load the OIDC parameters as normal settings into the database, That would also allow users to override them and not require magic here?